cpp/docs/DEVELOPER_GUIDE.md

```diff
@@ -964,21 +964,20 @@ this compound column representation of strings.
 
 ## Structs columns
 
-Structs are represented similarly to lists, except that they have multiple child data columns.
-The parent column's type is `STRUCT` and contains no data, but its size represents the number of 
-structs in the column, and its null mask represents the validity of each struct element. The parent 
-has `N + 1` children, where `N` is the number of fields in the struct. 
+A struct is a nested data type parametrized by ordered sequence of types, called fields. Unlike
+[Apache Arrow](https://arrow.apache.org/docs/format/Columnar.html#struct-layout), cuDF struct
+type only differentiate with other type by the field data type and the field order. Field name is
+not represented.
 
-1. A non-nullable column of `INT32` elements that indicates the offset to the beginning of each 
-   struct in each dense column of elements.
-2. For each field, a column containing the actual field data and optional null mask for all elements
-   of all the structs packed together.
-   
-With this representation, `child[0][offsets[i]]` is the first field of struct `i`, 
-`child[1][offsets[i]]` is the second field of struct `i`, etc.
+A structs column with `N` fields has `N` children. Each child is a column storing all the data
+of a single field packed column-wise, with an optional null mask. The parent column's type is
+`STRUCT` and contains no data, its size represents the number of structs in the column, and its
+null mask represents the validity of each struct element.
+
+With this representation, `child[0][10]` is the first field of struct `10`, `child[1][42]` is the second
+field of struct `42`, etc.
```

Comment:
<!--StartFragment-->With this representation, `child[0][10]` is the first field of struct `10`, `child[1][42]` is the second




    
```suggestion
With this representation, `child[0][10]` is row 10 of the first field of the struct, `child[1][42]` is row 42 of the second field of the struct.
```